packages/mobile/src/helpers/helpers.ts

```diff
@@ -1 +1,3 @@
 export { getQueryString } from './http/http';
+export { formatPrice } from './name-modifier/name-modifier';
```

Comment:
What "name" are we talking about? Please use some common and understandable names instead.
format, for instance

Reply:
changed